app/pages/expenses/goodsToSellOrUse/TaxiMinicabOrRoadHaulagePage.scala

```diff
@@ -27,8 +26,11 @@ case object TaxiMinicabOrRoadHaulagePage extends OneQuestionPage[Boolean] {
   override def toString: String = "taxiMinicabOrRoadHaulage"
 
   override def nextPageInNormalMode(userAnswers: UserAnswers, businessId: BusinessId, taxYear: TaxYear): Call =
-    routes.GoodsToSellOrUseAmountController.onPageLoad(taxYear, businessId, NormalMode)
+    cyaPage(taxYear, businessId)
```

Comment:
Looks like this shouldn't go to the CYA page

Reply:
good catch, reverted